app/controllers/api/v3/runs_controller.rb

```diff
@@ -4,9 +4,7 @@ class Api::V3::RunsController < Api::V3::ApplicationController
 
   before_action only: [:create] do
     # If an OAuth token is supplied, use it (and fail if it's invalid). Otherwise, upload anonymously.
-    if request.headers['Authorization'].present?
-      doorkeeper_authorize! :upload_run
-    end
+    doorkeeper_authorize! :upload_run if request.headers['Authorization'].present?
```

Comment:
I have no idea of the precedence Ruby puts on stuff like this, is this equivalent to
```ruby
doorkeeper_authorize!(:upload_run) if request.headers['Authorization'].present?
```
or
```ruby
doorkeeper_authorize!(:upload_run if request.headers['Authorization'].present?)
```
? Either way I don't like the ambiguity to humans, so let's be explicit when we have situations like this.

Reply:
Yeah, I agree that it looks cleaner when its more laid out, and the precedence might change.